packages/react-native-gesture-handler/scripts/mark-commonjs-build.js

```
@@ -0,0 +1,29 @@
+// Run after `bob build`: flips `IS_COMMONJS_BUILD` to `true` in the
+// `lib/commonjs` output so `src/cjsGuard.ts` can detect that the CommonJS
+// artifact ended up inside a React Native app bundle. See that file for why
+// this can't be decided at runtime.
+const fs = require('fs');
```

ESLint

[error] Expected "import" instead of "require()"

(import-x/no-commonjs)